File: submissions/sm_007_brijesh/week_3/day3/session_2/average_divisible_while.js
// function avg_divisible(arr) {
//     var sum = 0
//     var arr = [2, 3, 6, 8, 9, 4]

//     while (i < arr.length) {
//         i--
//         sum += arr[i]

//     }
//     var average = sum / arr.length
//     console.log(average)
// }


var limit = 10;
var num = 5;

console.log("Average: ", avg_divisible(limit, num))


//function
function avg_divisible(limit, num){
    var sum = 0;
    var count = 0;
    var i = 0;
    while(i <= limit){

        if(i % num === 0){
           // console.log(i)
            sum = sum + i; // sum of numbers divisible by num
            count++;
        }
        i++;
    }console.log(count)
    avg = sum/count;
    return avg;
}console.log(avg_divisible(30,5))

File: submissions/sm_010_dipanshu/week_13/day_4/session_1/average_divisible_while.py
limit = 30
num = 3

sum_of_factors = 0
count = 0
avg = 0

i = 1
while i <= limit:
    if i % num == 0:
        sum_of_factors += i
        count += 1
    i += 1

avg = sum_of_factors / count
print(avg)
